dsl/scripts/jobs.groovy

```diff
@@ -903,7 +941,9 @@ def jobs = [
         name: 'common/system-tests-lnl-mainnet',
         useScmDefinition: false,
         definition: libDefinition('pipelineCapsuleLNL()'),
-        parameters: lnlSystemTestsparams(),
+        parameters: lnlSystemTestsparams(
+            NODE_LABEL: 'general',
```

Comment:
What is general node? LNL pipeline needs very powerful server. It starts the network with 13 validators + 1 non validator